R/run_randomisation_analysis.R

```diff
@@ -10,10 +10,20 @@
 #' @param def_query character
 #' @param ...
 #'
+#' @return Returns the results from calling the server
+#' 
 #' @export
 #' @examples
 #' if(interactive()) {
-#'   b = BiodiverseR::basedata$new(name = "trial")
+#'   run_randomisation_analysis (
+#'     bd                 = basedata$new(cellsizes=c(500,500))
```

Comment:
Basedata object needs to be created and have one or more spatial or cluster analyses run before a randomisation will run.

And now I think more on these analysis methods, many are essentially internal and called from the basedata object.  That reduces the doc burden, although it is still good to document for future devs.

Reply:
Added the creation of the basedata object and the cluster analysis